### PR TITLE
#12 Access to test mappings from dialects tests

### LINKED ISSUE #12: Access to test mappings from dialetcs tests
The test code of the common part contains test resouces that are accessed via getResource() The files ware however not readable from the dialtects test code if the jar is packed.

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #12

### CODE REVIEW

File: doc/changes/changes_0.3.0.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Virtual Schema Common Document 0.3.0, released 2020-09-04
+
+## Summary
+
+Fixed bug #12
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes_0.3.0.md:5]: If it's a bugfix, you should change the minor version number

Comment (AnastasiiaSergienko) [reply doc/changes/changes_0.3.0.md:5]: And I think you don't need a summary here, it's redundant

**Review by AnastasiiaSergienko — CHANGES_REQUESTED** [review]

File: doc/changes/changes_0.3.0.md (hunk 1 above)
Comment (jakobbraun) [reply doc/changes/changes_0.3.0.md:5]: Well, it is a bug, but it's solution did change the api for the dynamodb-vs. Hence I changed a major version number. Should I only increase the minor version number in that case anyway?

**Comment by AnastasiiaSergienko** [comment]
We've also decided to make some small changes in the changes.md file for release robot. Check updated template if you want to use RR for release:

https://raw.githubusercontent.com/exasol/release-robot/feature/16_improve_validation_and_release_process/doc/user_guide/changes_file_template.md

**Review by AnastasiiaSergienko — APPROVED** [review]

**Pull request merged**